src/cli.ts

```diff
@@ -369,7 +381,7 @@ export function run(processArgv: readonly string[]): void {
       buildSoyOptions,
       async (argv) => {
         const config = loadConfig(argv);
-        assertNonNullable(config.soyOptions);
+        assertNonNullable(config, "soyOptions");
```

Comment:
Is this suggestion correct?

```suggestion
        assertNonNullableWithConfig(config, "soyOptions");
```

Reply:
Yes! Thank you!